[DLS] Reorgs DLS docs and add limits page (#13777)
cloudflare/cloudflare-docs: content/data-localization/limitations.md

---
title: Limitations
pcx_content_type: reference
weight: 8
meta:
  title: Limitations
---

# Limitations

There are some caveats and limitations when deploying Data Localization Suite features.

> Cloudflare is working hard to improve this offering and fill the gaps. If you have a specific feature requests, please contact your [Account Team](/support/contacting-cloudflare-support/).

Reviewer: ```suggestion
Cloudflare is working hard to improve this offering and fill the gaps. If you have a specific feature request, please contact your [Account Team](/support/contacting-cloudflare-support/).
```

Learn more about how it works in our [blog post](https://blog.cloudflare.com/geo-key-manager-how-it-works/).

## Regional Services

When using Regional Services, some caveats and limitations may apply.

For product-specific caveats, refer to [Cloudflare product compatibility](/data-localization/compatibility/) page.

The following features and protocols are not supported by Regional Services and might not work on regionalized hostnames:

- Unencrypted HTTP (plain text) requests
- [ICMP](https://www.cloudflare.com/learning/ddos/glossary/internet-control-message-protocol-icmp/)
- [Encrypted Client Hello (ECH)](/ssl/edge-certificates/ech/)
- [Orange-to-Orange (O2O)](/cloudflare-for-platforms/cloudflare-for-saas/saas-customers/how-it-works/)
- [Onion Routing (Tor)](/network/onion-routing/)

> Due to the fact that Regional Services leverages Spectrum in the background, [Spectrum limitations](/spectrum/reference/limitations/) apply.

Reviewer: ```suggestion
Since Regional Services leverages Spectrum in the background, [Spectrum limitations](/spectrum/reference/limitations/) apply.
```
